daemon/mgr/network.go

```diff
@@ -65,13 +65,32 @@ type NetworkManager struct {
 }
 
 // NewNetworkManager creates a brand new network manager.
-func NewNetworkManager(cfg *config.Config, store *meta.Store) (*NetworkManager, error) {
+func NewNetworkManager(cfg *config.Config, store *meta.Store, ctrMgr ContainerMgr) (*NetworkManager, error) {
```

Comment:
can we replace `ctrMgr` to `containerMgr`, because `ctrMgr` may be a little confused.

Reply:
I think both is ok, short name in function paramters